sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/PartitionedQueryExecutionInfoInternal.java

```diff
@@ -64,6 +66,13 @@ public void setQueryRanges(List<Range<PartitionKeyInternal>> queryRanges) {
         this.queryRanges = queryRanges;
     }
 
+    public HybridSearchQueryInfo getHybridSearchQueryInfo() {
```

Comment:
no side effects on getters.

Reply:
Updated the code the code to use lazy initialization, which only happens during the first access.